legacy/ui/legacy/src/main/java/com/fsck/k9/ui/settings/SettingsListFragment.kt

```diff
@@ -138,29 +138,40 @@ class SettingsListFragment : Fragment(), ItemTouchCallback {
                     icon = Icons.Outlined.Help,
                 )
 
-                when (fundingManager.getFundingType()) {
-                    FundingType.GOOGLE_PLAY -> {
-                        // TODO add funding action
-                    }
-
-                    FundingType.LINK -> {
-                        addUrlAction(
-                            text = getString(R.string.settings_list_action_support, appNameProvider.appName),
-                            url = getString(R.string.donate_url),
-                            icon = Icons.Outlined.Favorite,
-                        )
-                    }
-
-                    FundingType.NO_FUNDING -> {
-                        // no-op
-                    }
-                }
+                addFunding()
             }
         }
 
         itemAdapter.setNewList(listItems)
     }
 
+    private fun SettingsListBuilder.addFunding() {
+        when (fundingManager.getFundingType()) {
+            FundingType.GOOGLE_PLAY -> {
+                addIntent(
+                    text = getString(R.string.settings_list_action_support, appNameProvider.appName),
+                    icon = Icons.Outlined.Favorite,
+                    intent = FeatureLauncherActivity.getIntent(
+                        context = requireActivity(),
+                        target = FeatureLauncherTarget.Funding,
+                    ),
+                )
```

Comment:
We should get into the habit of hiding non-functional/incomplete features behind a feature flag so daily users are not exposed to it.